.github/workflows/build.yml

```diff
@@ -14,14 +14,31 @@ jobs:
       - name: "checkout repository"
         uses: "actions/checkout@v2"
       - name: "setup jdk 11"
-        uses: "actions/setup-java@v1"
+        uses: "actions/setup-java@v2"
         with:
+          distribution: "adopt"
           java-version: "11"
       - name: "build"
         run: "./gradlew build"
+      - name: "Archive test results"
```

Comment:
```suggestion
      - name: "archive test results"
```

Reply:
that's a straight copy + paste from adventure :p